tests/commands/DriveToWaypointCommandTest.cpp

```diff
@@ -0,0 +1,96 @@
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <cstdio>
+
+#include "../../src/commands/CommandBase.h"
+#include "../../src/commands/DriveToWaypointCommand.h"
+
+using namespace robot::types;
+using namespace std::chrono_literals;
+
+void assertApprox(double expectedThetaVel, double expectedXVel,
+                  const commands::command_t& actual, double angle = 1e-5,
+                  double vel = 1e-5) {
```

Comment:
Maybe clarify that `angle` and `vel` are error tolerances, not the actual angle and vel.